infra/gcp/terraform/k8s-infra-prow-build/prow-build/resources/monitoring/prometheus-main/prometheus.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: main
+  namespace: monitoring
+spec:
+## disable alerting for the time being
+#  alerting:
+#    alertmanagers:
+#      - namespace: monitoring
+#        name: alertmanager-operated
+#        port: web
+  version: v2.50.1
+  serviceAccountName: prometheus
+  podMonitorNamespaceSelector: {}
+  ruleSelector:
+    matchLabels:
+      prometheus: main
+  podMonitorSelector:
+    matchLabels:
+      prometheus: main
+  serviceMonitorNamespaceSelector: {}
+  serviceMonitorSelector:
+    matchLabels:
+      prometheus: main
+  resources:
+    requests:
+      cpu: 4
+      memory: 32Gi
+    limits:
+      cpu: 4
+      memory: 32Gi
+  replicas: 1
+  logLevel: info
+  logFormat: logfmt
+  retention: 7d
+  scrapeInterval: 15s
+  securityContext:
+    fsGroup: 0
+    runAsNonRoot: false
+    runAsUser: 0
+  storage:
+    volumeClaimTemplate:
```

Comment:
We should check if the default StorageClass on GKE is "good-enough" for Prometheus (e.g. not too expensive, solid performance, etc.).